README.md

```diff
@@ -66,6 +66,20 @@ When you use `Telescope neorg find_aof_project_tasks` you can pick an area of fo
 <video alt="find_aof_project_tasks" src="https://user-images.githubusercontent.com/81827001/158401841-9ca3a311-bac1-4733-9a6e-6125003d8a38.mov"></video>
 </details>
 
+Until the new GTD infrastructure arrives, you could collect all your todo items with a regex, for example:
+
+```lua
+-- Add the following function to your `~/.config/nvim/init.lua`:
+function get_todos(dir, states)
+     require('telescope.builtin').live_grep{cwd=dir}
+     vim.fn.feedkeys('^ *([*]+|[-]+) +[(]' .. states .. '[)]')
+end
+
+-- You can now use `:lua get_todos('~/notes', '[^x_]')` to get all todo items inside directory '~/notes' that are not yet done or cancelled.
```

Comment:
```suggestion
```
mapping is enough
if anything more then a command
we don't want users to have to call lua functions like this